src/codeu/chat/client/commandline/Chat.java

```diff
@@ -111,6 +118,32 @@ public void invoke(Scanner args) {
         System.out.println("    Exit the program.");
       }
     });
+    
+    
+    // INFO 
+    //
+    // Gives user information on the server 
+    // Added only for UpTime which returns the amount of time the server has been running for 
+    
+    panel.register("info", new Panel.Command() {
+    	  @Override
+    	  public void invoke(Scanner args) {
+    	    final ServerInfo info = context.getInfo();
+    	    final Time currentTime = Time.now(); 
+    	    if (info == null) {
+    	    	System.out.println("ERROR: Failed to retrieve server information"); 
+    	    } else {
+    	    	long upTime = currentTime.inMs() - info.getStartTime().inMs();
```

Comment:
optional: you might think about moving this logic into its own class, that way you can easily test the functionality to make sure your math is right